### ISSUE TITLE
[Feature]  Use Video thumbnail instead of Channel thumbnail for YouTube videos in widgets.

### ISSUE THREAD

**Issue by scapix** [jurialmunkey/skin.arctic.horizon#255]
Arctic Zephyr shows video thumbs instead of seeing same channel thumbnails. Would be nice to have same functionality in Horizon.

**Comment by jurialmunkey** [comment]
Youtube already shows thumbs for videos for me...

![Honeyview_screenshot193](https://user-images.githubusercontent.com/4618719/91251755-72334380-e79f-11ea-827c-2e3205c3ec98.jpg)

**Comment by scapix** [comment]
![8EA63855-E510-444B-A807-33E43456668F](https://user-images.githubusercontent.com/2369415/91352150-b8ad9000-e7e9-11ea-8fe0-c4da6df729e2.jpeg)
![BD39E2A6-01B5-4FCA-8AB7-75B25367A0F3](https://user-images.githubusercontent.com/2369415/91352159-ba775380-e7e9-11ea-9092-5cc36f0eb43a.jpeg)

Hmm Zephyr2 vs Horizon both using (Small) Landscape viewmode

**Comment by scapix** [comment]
fixed in 0.5.44

**Comment by scapix** [comment]
Hmm. It went back to showing channel art 😒

**Comment by jurialmunkey** [comment]
I don't think this is a skin issue. Sounds more like an issue with the youtube plugin if it keeps changing back and forth because I didn't make any changes there.

**Comment by scapix** [comment]
Yes, you are right. It was because "Use (Channel) Fanart" setting was turned on in YouTube plugin. In Horizon skin it also switches to Channel Thumbnail for focused when browsing plugin content (unfocused items are showing video thumb). It "worked" in Zephyr2 because of different "Image_Landscape" variable evaluation.

**Comment by jurialmunkey** [comment]
That focus issue sounds strange, I'll try to have a look and see if I can find what the issue is now that we know what setting it is related to.
